packages/ui-react/src/hooks/useFilteredContent.ts

```diff
@@ -0,0 +1,73 @@
+import type { Content } from '@jwp/ott-common/types/config';
+import { Breakpoint, getScreenSize } from '@jwp/ott-ui-react/src/hooks/useBreakpoint';
+
+const DEVICE_FILTER_LABELS: Record<string, string> = {
+  mobile: 'mobile',
+  tablet: 'tablet',
+  desktop: 'desktop',
+} as const;
+
+const COUNTRIES: Record<string, string> = {
+  DE: 'Germany',
+  FR: 'France',
+  NL: 'Netherlands',
+  RO: 'Romania',
+} as const;
+
+const TIMEZONES: Record<string, string> = {
+  'Europe/Berlin': 'DE',
+  'Europe/Paris': 'FR',
+  'Europe/Amsterdam': 'NL',
+  'Europe/Bucharest': 'RO',
+} as const;
+
+const getDeviceType = () => {
+  const breakpoint = getScreenSize();
+  const isMobile = breakpoint < Breakpoint.md;
+  const isTablet = breakpoint >= Breakpoint.md && breakpoint < Breakpoint.lg;
+  const isDesktop = breakpoint >= Breakpoint.lg;
+
+  return { isMobile, isTablet, isDesktop };
+};
+
+const getCountryByTimezone = () => {
+  const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+
+  if (!timezone || !(timezone in TIMEZONES)) {
+    return undefined;
+  }
+
+  const countryCode = TIMEZONES[timezone];
+  return COUNTRIES[countryCode];
+};
+
+const getCurrentDay = () => new Date().toLocaleString('en-US', { weekday: 'long' });
+
+const filterDefaultContent = (item: Content) => !item?.filterTags?.length;
+
+const filterContentByDevice = (item: Content) => {
+  const { isMobile, isTablet, isDesktop } = getDeviceType();
+
+  if (item?.filterTags?.includes(DEVICE_FILTER_LABELS.mobile) && isMobile) return true;
+  if (item?.filterTags?.includes(DEVICE_FILTER_LABELS.tablet) && isTablet) return true;
+  if (item?.filterTags?.includes(DEVICE_FILTER_LABELS.desktop) && isDesktop) return true;
+};
+
+const filterContentByWeekDay = (item: Content) => {
+  const currentDay = getCurrentDay();
+  return item.filterTags?.includes(currentDay);
+};
+
+const filterContentByCountry = (item: Content) => {
+  const country = getCountryByTimezone();
+  return country && item.filterTags?.includes(country);
```

Comment:
We receive `filterTags` as a string, so I think we should first split it by comma.

Reply:
`includes` also works for strings, but I updated to use `split` 👍🏻